### ISSUE TITLE
Improvement - Add frontend module for hamburger menu

### ISSUE THREAD

**Issue by contaoacademy** [dklemmt/contao_dk_mmenu#72]
It would be helpful if there is a "hambuerger element" as a Contao frontend module.

Maybe [mburger webcomponent](https://github.com/FrDH/mburger-webcomponent/) can be used?
see: https://mmenujs.com/mburger/

**Comment by fritzmg** [comment]
> Maybe [mburger webcomponent](https://github.com/FrDH/mburger-webcomponent/) can be used?

I don't think we can/should use that as this is a web component and its JS is a module.
